avpipe_log.go

```diff
@@ -0,0 +1,73 @@
+package avpipe
+
+import (
+	"sync"
+
+	"github.com/modern-go/gls"
+
+	elog "github.com/eluv-io/log-go"
+)
+
+// logWrapper is used to wrap the standard log-go logger to include the handle of the AVPipe job in
+// question, if it is known
+type logWrapper struct {
+	log *elog.Log
+}
+
+func (l *logWrapper) Trace(msg string, fields ...interface{}) {
+	fields = append(fields, logHandleIfKnown()...)
+	l.log.Trace(msg, fields...)
+}
+
+func (l *logWrapper) Debug(msg string, fields ...interface{}) {
+	fields = append(fields, logHandleIfKnown()...)
+	l.log.Debug(msg, fields...)
+}
+
+func (l *logWrapper) Info(msg string, fields ...interface{}) {
+	fields = append(fields, logHandleIfKnown()...)
+	l.log.Info(msg, fields...)
+}
+
+func (l *logWrapper) Warn(msg string, fields ...interface{}) {
+	fields = append(fields, logHandleIfKnown()...)
+	l.log.Warn(msg, fields...)
+}
+
+func (l *logWrapper) Error(msg string, fields ...interface{}) {
+	fields = append(fields, logHandleIfKnown()...)
+	l.log.Error(msg, fields...)
+}
+
+func (l *logWrapper) Fatal(msg string, fields ...interface{}) {
+	fields = append(fields, logHandleIfKnown()...)
+	l.log.Fatal(msg, fields...)
+}
+
+var log = logWrapper{log: elog.Get("/eluvio/avpipe")}
```

Comment:
How about just `/avpipe` here - we were looking at eliminating some of the other `/eluvio` prefixes since they just add bytes and make config more lengthy.

Reply:
Yeah that's fine by me. It's still unambiguous.